### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by chaoalexander** [ageron/handson-ml3#76]
**Describe the bug**
In the third edition of the book, in Chapter 11 of the Max Norm Regulation, in the last paragraph, you write, 'The max_norm() function has an axis argument that defaults to 0.' A Dense layer usually has weights of shape [number of inputs, number of neurons], so using axis=0 means that the max-norm constraint will apply independently to each neuron’s Weight vector ", which looks similar to 'https://www.tensorflow.org/api_docs/python/tf/keras/constraints/MaxNorm'. The explanation given is different, axis=0 represents the input dimension
